migrations/Version202103240929581465_qtiItemPci.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\qtiItemPci\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+
+final class Version202103240929581465_qtiItemPci extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Added support for review mode for Audio Recording interaction';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $registry = (new PciModel())->getRegistry();
```

Comment:
**nitpick:** Missing use clause for the class PciModel.
**suggestion:** Could you add these:
```php
use Doctrine\Migrations\Exception\IrreversibleMigration;
use oat\qtiItemPci\model\PciModel;
use oat\qtiItemPci\scripts\install\RegisterPciAudioRecording;
```

Reply:
done!